desktop.css

```diff
@@ -0,0 +1,378 @@
+/* ==================== Removed elements ================= */
+
+.menu,
+.geometry-shape1,
+.headline-image {
+  display: none !important;
+}
+
+/* ==================== General styles ================= */
+
+body {
+  background: none;
+}
+
+.primary-text {
+  font-size: 110px;
+  line-height: 100%;
+}
+
+.primary-button {
+  max-width: 254px;
+  height: 70px;
+}
+
+.max-width {
+  max-width: 1440px;
+}
+
+.dark-button {
+  max-height: 51px;
+  max-width: 212px;
+  background: white;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+  transition: all 0.3s;
+}
+
+.dark-button i {
+  background: white;
+  color: #1a2236;
+  transition: all 0.3s;
+}
+
+.dark-button:hover {
+  background: #1a2236;
+  color: white;
+  border: 1px solid white;
+}
+
+.dark-button:hover i {
+  background: #1a2236;
+  color: white;
+}
+
+.dark-button:active {
+  border: 2px solid white;
+}
+
+/* ==================== header styles ==================== */
+
+header {
+  height: 100px;
+  padding: 38px 0;
+  justify-content: center;
+  box-shadow: 0 0.5px 0 #dddfe6;
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+  gap: 50px;
+}
+
+.nav-links a {
+  font-size: 16px;
+  line-height: 24px;
+  color: #1a2236;
+  background:
+    linear-gradient(to right, transparent, transparent),
+    linear-gradient(to right, black, black, black);
+  background-size: 100% 0.1em, 0 0.1em;
+  background-position: 100% 100%, 0 100%;
+  background-repeat: no-repeat;
+  transition: background-size 400ms;
+}
+
+.nav-links a:hover,
+.nav-links a:focus {
+  background-size: 0 0.1em, 100% 0.1em;
+}
+
+/* ==================== headline styles ==================== */
+
+.headline {
+  display: grid;
+  align-items: center;
+  width: 100%;
+  grid-template-columns: 50% 50%;
+  margin-bottom: 5%;
+}
+
+.info {
+  height: 100%;
+  max-width: 100%;
+  justify-content: center;
+  align-items: flex-end;
+  text-align: left;
+  margin: 0 5%;
+  gap: 50px;
+  background: url(Assets/Images/image-dots.svg);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+.info p,
+.info h1,
+.info button {
+  max-width: 484px;
+}
+
+.headline-image2 {
+  display: block;
+  width: 100%;
+}
+
+/* ==================== Works styles ==================== */
+
+.works {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 32.291% 23.263% 12.152% 20.069% 12.222%;
+  grid-template-rows: 33.941% 4.866% 11.557% 4.866% 11.557% 6.082% 4.587% 6.057% 7.542% 8.878%;
+  height: 150vh;
+  max-height: 1644px;
+  gap: 0;
+}
+
+.works-title {
+  grid-area: 1/1/2/2;
+  background-image: url(assets/Images/image-dots.svg);
```

Comment:
- [Optional] Kindly, your images may be displayed properly on your local machine, however, it will not in most environments, because most environments are case-sensitive on the static directory, so updated it to display your image properly, I will suggest you to use only small letters for static directory names instead of updating the URL to follow best practices👌👌,